src/components/Organisation/Banner/index.tsx

```diff
@@ -13,33 +20,68 @@ import { useDispatch, useSelector } from 'react-redux';
 import _debounce from 'lodash/debounce';
 import { editOrganisation } from 'actions/organisation';
 import { RootState } from 'reducers';
+import { ReactComponent as EditIcon } from 'assets/illustrations/icons/edit.svg';
+import Background from 'assets/illustrations/profile/pp-bg.png';
 import styles from './index.module.scss';
 import OrganisationSocialModal from './OrganisationSocialModal';
-import { useUpdateOrganisation } from './hooks';
+import {
+  useUpdateOrganisation,
+  useUpdateOrgPic,
+  useUpdateOrgCoverPic,
+} from './hooks';
 
 interface IBanner {
   organisation: IOrganisation;
 }
 
+interface IFile {
+  id: string;
+  file: File;
+}
+
 const Banner: FC<IBanner> = ({ organisation }) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const inputRefCover = useRef<HTMLInputElement>(null);
   const dispatch = useDispatch();
   const isEditable = useSelector((state: RootState) => state.org.isEditable);
   const user = useSelector((state: RootState) => state.user.user);
 
   const [nameLocal, setNameLocal] = useState(organisation.name ?? 'Polkadot');
   const [website, setWebsite] = useState(organisation.website ?? '');
   const [open, setOpen] = useState(false);
+  const [orgLogoFileData, setOrgLogoFileData] = useState<IFile | null>(null);
+  const [orgCoverFileData, setCoverLogoFileData] = useState<IFile | null>(null);
 
   const updateOrganisation = useUpdateOrganisation(organisation.organisationId);
-
+  const updateOrgPicture = useUpdateOrgPic(organisation.organisationId);
+  const updateCoverOrgPicture = useUpdateOrgCoverPic(
+    organisation.organisationId
+  );
+  let updatedOrgCoverUrl = null;
+  console.log(organisation);
   const payload = {
     name: nameLocal,
     description: organisation.description,
     whitepaper: organisation.whitepaper,
     website,
   };
+  const formData = new FormData();
+  useEffect(() => {
+    if (orgLogoFileData) {
+      formData.append('file', orgLogoFileData.file);
+      formData.append('type', 'logo');
+      updateOrgPicture.mutate(formData);
+    }
+    if (orgCoverFileData) {
+      formData.append('file', orgCoverFileData.file);
+      formData.append('type', 'cover');
+      updatedOrgCoverUrl = URL.createObjectURL(orgCoverFileData.file);
+      updateCoverOrgPicture.mutate(formData);
+    }
+  }, [orgLogoFileData, orgCoverFileData]);
```

Comment:
I think since we are having a "Save" button, we can use it to save everything?
And when both `orgLogoFileData` and `orgCoverFileData` are there, if we change any one of the two then it'll call the API for both of them.